echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/scheduler/actions/pipeline/impl/MissedPipelineTriggerCompensationJob.groovy

```diff
@@ -197,11 +198,16 @@ class MissedPipelineTriggerCompensationJob implements ApplicationListener<Contex
         return
       }
 
-      def expr = new CronExpression(trigger.cronExpression)
-      expr.timeZone = dateContext.timeZone
+      try {
+        CronExpression expr = new CronExpression(trigger.cronExpression)
+        expr.timeZone = dateContext.timeZone
 
-      if (missedExecution(expr, lastExecution, dateContext.triggerWindowFloor, dateContext.now, pipeline)) {
-        pipelineInitiator.call(pipeline)
+        if (missedExecution(expr, lastExecution, dateContext.triggerWindowFloor, dateContext.now, pipeline)) {
+          pipelineInitiator.call(pipeline)
+        }
+      } catch (ParseException e) {
+        log.error("Error parsing cron expression (${trigger.cronExpression}) for pipeline ${pipeline.id}")
```

Comment:
log.error("Error parsing cron expression (${trigger.cronExpression}) for pipeline ${pipeline.id}", e)

Reply:
Logging a stacktrace here for a java.text.ParseExpression isn't helpful, no longer doing so was deliberate. Having the cron expression that triggers it on the other hand is useful.